configuration-rendering-controller/README.md

```diff
@@ -209,7 +209,174 @@ configs). For now, it makes sense to keep our current logic as is to reduce time
 
 ### Defining scope of configs to render
 
-TBD
+```yaml
+apiVersion: configuration.giantswarm.io/v1alpha1
+kind: ManagementClusterConfiguration
+metadata:
+  name: gauss-configuration
+  namespace: giantswarm
+spec:
+  sources:
+    flux:
+      service:
+        url: source-controller.flux-giantswarm.svc
+      gitRepository:
+        name: giantswarm-config
+        namespace: flux-giantswarm
+  encryption:
+    sops:
+      keysDirectory: /mnt/sops
+    vault:
+      secretRef:
+        name: gauss-vault-configuration
+        namespace: giantswarm
+  target:
+    namespace: giantswarm-configuration
+  configuration:
+    cluster:
+      name: gauss
+    applications:
+      regexMatchers:
+        - ".+"
+status:
+  inventory:
```

Comment:
Hmm, is this `inventory` object not too big? We already have something in the range of hundreds of apps, I think? This means this will be huge, hard to work with and changing all the time. Maybe this level of details should be in metrics only?

Reply:
Just metrics is also fine, tho some representation / overview would be nice on the resources. Its simply faster to check than going to Prometheus / Grafana. Tho we could also set up a nice dashboard.

It also came to my mind to generate report CRs, like Trivy or Kyverno.